features/filmStrip/components/VideoThumbnail.js

```diff
@@ -106,16 +109,22 @@ class VideoThumbnail extends Component {
      */
     render() {
         let participant = this.props.participant;
+        let largeVideo = this.props.largeVideo;
         let streams = this.getMediaStreams();
         let renderAudio =
             streams.audio
-                && !this.props.audioMuted
-                && !this.props.audioTrack.isLocal();
-        let renderVideo = streams.video && !this.props.videoMuted;
+            && !this.props.audioMuted
+            && !this.props.audioTrack.isLocal();
+        let renderVideo = 
+            streams.video
+            && !this.props.videoMuted 
+            && (!participant.videoStarted 
+                || (participant.videoStarted 
+                    && participant.id !== largeVideo.onStageParticipantId));
```

Comment:
That's something I've fixed already during my review so it's merely a note, not a call for action:

````
    !participant.videoStarted 
    || (participant.videoStarted 
        && participant.id !== largeVideo.onStageParticipantId)
````
is the same as:
````
!participant.videoStarted || participant.id !== largeVideo.onStageParticipantId
````